code/frameworks/web-components-webpack5/package.json

```diff
@@ -62,7 +62,7 @@
     "typescript": "~4.9.3"
   },
   "peerDependencies": {
-    "lit-html": "^1.4.1 || ^2.0.0",
+    "lit-html": "^2.0.0",
```

Comment:
Does this count as a “full blown” dependency due to its use in https://github.com/storybookjs/storybook/blob/9e6a958367d5ab9159b210d1d4d67598909170b8/code/renderers/web-components/src/render.ts#L7 ?

Reply:
I think peer dependencies are still OK in this case. Will double check with the team.

Reply:
Usually you either have a `dependency` or a `peerDependency`+`devDependency`.

...having a `peerDependency`+`dependency` doesn't make a lot of sense.